lib/krane/resource_deployer.rb

```diff
@@ -49,7 +49,8 @@ def predeploy_priority_resources(resource_list, predeploy_sequence)
       predeploy_sequence.each do |resource_type, attributes|
         matching_resources = resource_list.select do |r|
           r.type == resource_type &&
-          (!attributes[:group] || r.group == attributes[:group])
+          (!attributes[:group] || r.group == attributes[:group]) &&
+          r.predeployed?
```

Comment:
We check for the predeployment status of CustomResources when we build the predeploy_sequence. Could we also get away with only including the predeployable "additional resources" (deployments, services, etc) therein?

```suggestion
```